workflow-templates/im-test-k6-operator.yml

```diff
@@ -226,6 +230,8 @@ jobs:
           parallelism: ${{ github.event.inputs.parallelism }}
           kube-config-file-path: ${{ steps.kube-config.outputs.kube-config-file }}
           retention-in-days: ${{ env.RETENTION_IN_DAYS }}
+          loop-count: ${{ github.event.inputs.loop-count }}
+          # loop-duration: ${{ env.LOOP_DURATION }} # TODO: uncomment if you want to customize loop duration
```

Comment:
If this is the only place we're using the `LOOP_DURATION` env variable, and we have to uncomment it in two places, it might make more sense to not have the env variable and just throw the value directly in here. That way we only have to uncomment one area.

Reply:
I will update the env variable to not uncomment, but just a todo next to it.  Then there will be only one place to uncomment.

Reply:
I updated to address this.